zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/OperateElasticsearchExporter.java

```diff
@@ -175,15 +186,13 @@ private ExportBatchWriter createBatchWriter() {
   private static final class ElasticsearchRecordFilter implements Context.RecordFilter {
     private static final List<ValueType> VALUE_TYPES_2_IMPORT =
         List.of(
-            PROCESS, VARIABLE, PROCESS_INSTANCE
+            PROCESS, VARIABLE, PROCESS_INSTANCE, USER_TASK, INCIDENT
```

Comment:
Note: Incident is added here just because I missed it when I merged the PR for 18913 and I didn't think it was worth an entirely separate PR